config/routes.rb

```diff
@@ -11,6 +11,7 @@
           put "undo_publish"
           put "undo_rejection"
           put "undo_close"
+          put "list_artwork"
```

Comment:
Yeah, no need to change it. I see how this also acts as an operation on a submission, as far as Convection is concerned and actually does mutate the submission data a bit. The created artwork is more of a side-effect of how a submission is listed rather than structured/exposed as a first-class association.